pkg/420f1a95-2058-4f30-bf54-3d06eb49b2e3/package.lp

```diff
@@ -0,0 +1,84 @@
+ (pad 81bee4e7-f9fa-4bb7-a935-de82f8ec72f9 (name "6"))
+ (footprint ffc812a0-a563-46b2-97c9-af0d022f0c60
+  (name "default")
+  (description "")
+  (pad 9dc3bdde-2818-4f92-904e-748e2e5b8113 (side top) (shape rect)
+   (position -2.75 2.5) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (pad 463f0037-c13d-4cb4-a0f6-0293a389876f (side top) (shape rect)
+   (position -2.75 0.0) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (pad 0906829d-a16a-4da4-a4f5-16a39555cf56 (side top) (shape rect)
+   (position -2.75 -2.5) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (pad b2e833a7-7390-4362-9568-5ca9c898b242 (side top) (shape rect)
+   (position 2.75 2.5) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (pad 949201b6-de70-48b2-94d1-c7748c64cb56 (side top) (shape rect)
+   (position 2.75 0.0) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (pad 81bee4e7-f9fa-4bb7-a935-de82f8ec72f9 (side top) (shape rect)
+   (position 2.75 -2.5) (rotation 0.0) (size 2.5 1.2) (drill 0.0)
+  )
+  (polygon 86d3724e-4afb-4663-9d0d-8cfeba95e13f (layer top_documentation)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -1.7 4.4) (angle 0.0))
+   (vertex (position 1.7 4.4) (angle 0.0))
+   (vertex (position 1.7 -4.4) (angle 0.0))
+   (vertex (position -1.7 -4.4) (angle 0.0))
+   (vertex (position -1.7 4.4) (angle 0.0))
+  )
+  (polygon 09f9e6ad-e16d-4b54-8881-40c6b7bd508f (layer top_placement)
+   (width 0.2) (fill false) (grab_area false)
+   (vertex (position -3.9 3.4) (angle 0.0))
+   (vertex (position -1.9 3.4) (angle 0.0))
+   (vertex (position -1.9 4.6) (angle 0.0))
+   (vertex (position 1.9 4.6) (angle 0.0))
+   (vertex (position 1.9 3.4) (angle 0.0))
+  )
+  (polygon 0f1e38d5-743c-4472-912f-58197c7047a6 (layer top_placement)
+   (width 0.2) (fill false) (grab_area false)
+   (vertex (position 1.9 -3.4) (angle 0.0))
+   (vertex (position 1.9 -4.6) (angle 0.0))
+   (vertex (position -1.9 -4.6) (angle 0.0))
+   (vertex (position -1.9 -3.4) (angle 0.0))
+  )
+  (polygon 281e45a6-38cc-4240-bd04-5514979333a6 (layer top_documentation)
+   (width 0.1) (fill false) (grab_area false)
+   (vertex (position -0.75 1.75) (angle 0.0))
+   (vertex (position 0.75 1.75) (angle 0.0))
+   (vertex (position 0.75 -1.75) (angle 0.0))
+   (vertex (position -0.75 -1.75) (angle 0.0))
+   (vertex (position -0.75 1.75) (angle 0.0))
+  )
+  (stroke_text 5e3b8138-2026-4d9c-94b9-3a0c7097a188 (layer top_names)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center bottom) (position 0.0 5.225) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{NAME}}")
+  )
+  (stroke_text be3110e9-e6fc-454c-8721-eec042288076 (layer top_values)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center top) (position 0.0 -5.225) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{VALUE}}")
+  )
+  (stroke_text 86e43e68-ddf7-4a2c-a276-3e1d72be329d (layer brd_documentation)
```

Comment:
The text should be on top documentation:

```suggestion
  (stroke_text 86e43e68-ddf7-4a2c-a276-3e1d72be329d (layer top_documentation)
```

Reply:
Fixed!